lib/ui/settings/interactor/settings_state.dart

```diff
@@ -3,7 +3,7 @@ part of 'settings_bloc.dart';
 @freezed
 class SettingsState with _$SettingsState {
   const factory SettingsState({
-    @Default(PageState.initial) PageState pageState,
+    @Default(PageState.success) PageState pageState,

```

Comment:
im not sure this change should be here. we init our states with initial

Reply:
I wanted to express that "initial" and "success" are the same. 

Would it be better to indicate that by going from loading back to initial?

Not sure how to do that... can be future fix